integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/test/pagefactory/PageOptionsTest.java

```diff
@@ -23,14 +23,27 @@
 
 import eu.tsystems.mms.tic.testframework.AbstractTestSitesTest;
 import eu.tsystems.mms.tic.testframework.core.pageobjects.testdata.PageWithCheckTimeout;
+import eu.tsystems.mms.tic.testframework.core.pageobjects.testdata.PageWithExistingElement;
 import eu.tsystems.mms.tic.testframework.core.pageobjects.testdata.PageWithPageOptions;
 import eu.tsystems.mms.tic.testframework.testing.AssertProvider;
 import eu.tsystems.mms.tic.testframework.testing.PageFactoryProvider;
 import org.openqa.selenium.WebDriver;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 public class PageOptionsTest extends AbstractTestSitesTest implements PageFactoryProvider, AssertProvider {
 
+    /**
+     * This beforeClass is needed to initialize the used page factory implementation if it's not done at this point.
+     *
+     * (Currently ResponsivePageFactory)
+     */
+    @BeforeClass
+    public void pageOptionsInitializer() {
+        WebDriver webDriver = getClassExclusiveWebDriver();
+        PAGE_FACTORY.createPage(PageWithExistingElement.class, webDriver);
```

Comment:
What made this addition necessary c.p. before? (Or was it necessary-yet-absent even before?)

Reply:
See the comment in `ResponsiveClassFinder` (time problem). The tests in that class verify the timeout of page checks. If that tests are called before all other they would fail.

Reply:
Ach, I see! It's basically a dummy call so we "pre-heat" the reflections.
... Maybe create a dedicated method for that? (It can internally just call `createPage` and discard the result with an explicit `@SuppressWarnings("unused")`.)